### ISSUE TITLE
Incorrect crafting recipe for stained glass

### ISSUE THREAD

**Issue by Wuzzy2** [minetest-mods/homedecor_modpack#373]
The following crafting recipe is wrong:
Stained glass from glass pane and dye (probably wrong itemstring, should be `something:something_flat` or so)

**Comment by VanessaE** [comment]
A lot of the recipes are wrong now, have been since January when hardware colorization went in -- however, any updates to correct those errors have to wait until 0.4.17 goes out, because at that point I'll be able to re-activate the old some-item-plus-any-dye(s) recipes.

**Comment by Wuzzy2** [comment]
https://github.com/bell07/minetest-qa_block

This mod helps you in finding recipe duplicates.

The `xpanes` thing is actually simple: Just look up the stained glass in the crafting guide and you will see what's wrong.
The corrent itemstring is something with `_flat`.

**Comment by VanessaE** [comment]
Finally got a chance to look into this.  Both programmed recipes seem to work fine for me:
![image](https://user-images.githubusercontent.com/1733537/29244154-3cd8f1a2-7f7f-11e7-8865-b35812d048ab.png)
The middle item can be either xpanes:pane (as pictured), or default:glass.  Looks like the former is an old recipe from some time back, so I've removed it and put one in its place for "xpanes:pane_flat" (yields 3 stained glass).  I've added recipes for the two cottages glass panes also.

6349c538247a578f8f46da9359fc41b7ecee6647
